### DISCUSSION TITLE
Shrinking text in the overflow area

### DISCUSSION THREAD

**Discussion by TGayant** [ShapeCrawler/ShapeCrawler#181] [Q&A]
On my pptx template, I enable text reduction if it overflows from its area, but once the pptx is generated the reduction is not applied. Is it supported?
Thank you

**Comment by AdamShakhabov** [comment]
Hello @TGayant. 

If I correctly understood, the question of whether method to enable "Shrink text on overflow", which discussed in #165 issue. It is not added yet.

Please, let me know if your question is other.

**Comment by AdamShakhabov** [comment]
Looks like the question is duplication of [**Does ShapeCrawler support shrink text on overflow in paragraphs?**](https://github.com/ShapeCrawler/ShapeCrawler/discussions/163)
